changelogs/api-management-service.md

```diff
@@ -4,16 +4,16 @@
 
 ### New features, fixes, and improvements
 
-1. We have introduced new Azure Advisor recommendations:
-    - Use Azure AD authentication instead of gateway token for Configuration API
+1. We have introduced new Azure Advisor recommendations
+    -  Use Azure AD authentication instead of gateway token for Configuration API
     - Identity self-hosted gateways using gateway tokens that will expire in < 7 days
-    - Identity validate-jwt policy usage with insecure key size   
-2. We have removed support for the [retired set-graphql-resolver policy](https://learn.microsoft.com/en-us/azure/api-management/set-graphql-resolver-policy) which had already been replaced with GraphQL API resolvers.
-3. We have fixed an issue where the OpenAPI would fail if the parameter names where empty.
-4. We have fixed an issue where nested objects were being overridden when import OpenAPI was used.
-5. We have fixed an issue where the FixOpenApi parameter schema can now import complex schemas.
-6. We have fixed a bug where validate-azure-token policy was not properly using expressions to resolve tenant-id.
+    - Identity validate-jwt policy usage with insecure key size
+2. We have fixed an issue where the OpenApi would fail if the parameter names where empty.
```

Comment:
Markdown lets you use `1.` for every numbered item and will automatically number correctly for you. That means you can more easily make changes to the numbered list without having to update the numbering itself.